packages/flutter/lib/src/animation/tween.dart

```diff
@@ -55,6 +55,16 @@ class _ChainedEvaluation<T> extends Animatable<T> {
 }
 
 /// A linear interpolation between a beginning and ending value.
+///
+/// [Tween] is useful if you want to interpolate across a range.
+///
+/// To use a [Tween] object with an animation, call the [Tween] object's `animate()` method and
```

Comment:
This is missing the most important part - initialize the Tween with the desired range.

Also, it's common to use a specific Tween (like ColorTween) instead of using the unspecialized Tween class.